src/test/scala/scynamo/ScynamoCodecProps.scala

```diff
@@ -46,6 +46,11 @@ class ScynamoCodecProps extends Properties("ScynamoCodec") {
     decodeAfterEncodeIsIdentity(value)
   }
 
+  propertyWithSeed("decode.encode === id (instant @@ ttl)", propertySeed) =
+    Prop.forAll(Gen.calendar.map(_.toInstant).map(tag[TTL][Instant](_))) { value =>
```

Comment:
This won't work, you need to truncate to seconds

Reply:
Truncating in https://github.com/moia-oss/scynamo/pull/181/commits/713f06ced2add5656978f79b6fad7e367fbc1723.